ui/accordion-collapsible-panel.reel/accordion-collapsible-panel.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @module ui/collapsible-panel.reel
+ * @requires montage/ui/component
+ */
+var Component = require("../component").Component;
+
+/**
+ * @class AccordionCollapsiblePanel
+ * @extends Component
+ *
+ * The internal panel inside Accordion to display header abd body
+ */
+exports.AccordionCollapsiblePanel = Component.specialize(/** @lends AccordionCollapsiblePanel# */ {
+    constructor: {
+        value: function CollapsiblePanel() {
+            this.super();
+        }
+    },
+
+    /**
+     * The parent Accordion component contains this.
+     * @type {Accordion}
+     */
+    accordion: {
+        value: null
+    },
+
+    /**
+     * Is the collapsible panel expanded or collapsed
+     * @type {boolean}
+     */
+    isExpanded: {
+        set: function (value) {
+            this._isExpanded = value;
+            this.needsDraw = true;
+        },
+        get: function () {
+            return this._isExpanded;
+        }
+    },
+
+    enterDocument: {
+        value: function (firstTime) {
+            if (firstTime) {
+                var header = this.templateObjects.owner.element.children[0];
+                header.addEventListener("mouseup", this, false);
```

Comment:
@dharcourt @thibaultzanini ic, yeah, I think I will change to use press-composer, it can't be an button since the header is pass in from outside and it can be any component.